packages/colorpickers/src/styled/common/StyledRange.ts

```diff
@@ -60,24 +60,38 @@ const trackLowerStyles = (styles: string, modifier = '') => {
   `;
 };
 
-const colorStyles = (props: IStyledRangeProps & ThemeProps<DefaultTheme>) => {
-  const thumbBackgroundColor = getColorV8('neutralHue', 100, props.theme);
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  const thumbBackgroundColor = getColor({ theme, variable: 'background.default' });
```

Comment:
@lucijanblagonic maybe just me, but I'm finding these dark mode thumbs very difficult to see. Somehow I don't have the same problem with shadowed white over white background.

Reply:
+1, feels like an accessibility concern for sure.

Reply:
@steue @lucijanblagonic For clarity, we're referring to this:

![Screenshot 2024-07-17 at 7 40 13 AM](https://github.com/user-attachments/assets/a94b3f1b-a6da-4b7a-85cc-fa5251f96811)

Reply:
great observation yall! curious about your thoughts of bumping up the contrast by setting the thumbs to `bg/raised` with a -100 offset? (looks brighter)
<img width="682" alt="image" src="https://github.com/user-attachments/assets/97ef5957-0989-49fb-8bd3-db53bc495f27">

Reply:
> curious about your thoughts of bumping up the contrast by setting the thumbs to bg/raised with a -100 offset? (looks brighter)

@steue for me, that color tweak works great 🙌

Reply:
Updated. ✅ 

![Screenshot 2024-07-19 at 1 29 57 PM](https://github.com/user-attachments/assets/6741a6c4-b451-49e1-94d4-1bcfea752c7a)
![Screenshot 2024-07-19 at 1 29 44 PM](https://github.com/user-attachments/assets/d548ba55-cc47-485d-97b0-54bc3d361ad4)